### PR TITLE
Fixes for Python 3.6

### PR DESCRIPTION
**Pull request by BryanCutler**
Fixed imports causing problems with Python 3.6. Import in the form of `import text_extensions_for_pandas.io.bert as bert` end up causing a circular reference due to later imports also referencing `io` which has not been initialized yet. Using the form `from text_extensions_for_pandas.io import bert` avoid this.

### CODE REVIEW

File: text_extensions_for_pandas/io/test_bert.py [hunk 1]
```diff
@@ -32,6 +34,7 @@
 )
 
 
+#@pytest.mark.skipif(sys.version_info[:2] <= (3, 6), reason="BertModel segfault")
```
Comment (BryanCutler) [inline text_extensions_for_pandas/io/test_bert.py:37]: I'm getting a segfault locally when constructing the `BertModel`, maybe an env thing so see what happens in travis

**Comment by BryanCutler** [comment]
cc @kmh4321 @frreiss

File: text_extensions_for_pandas/io/test_bert.py (hunk 1 above)
Comment (BryanCutler) [reply text_extensions_for_pandas/io/test_bert.py]: yup, looks like it's just my env :confused:

**Review by kmh4321 — APPROVED** [review]
Looks great! I also encountered the circular imports causing this issue but your solution is better than my proposed solution of relative imports. thanks!

**Comment by BryanCutler** [comment]
Great, thanks for checking @kmh4321 . This also adds Python 3.6 to travis CI to make sure we don't break the imports again. I'll go ahead and merge.

**Comment by frreiss** [comment]
LGTM. I notice that this change set leaves `env.sh` unchanged, which means we will be developing with Python 3.7. I think that's ok for now, since Travis will be testing with 3.6.

**Pull request merged**